registrazione-sito-primario-pubblica-amministrazione.rst

```diff
@@ -0,0 +1,9 @@
+**Registrazione sito primario Pubblica amministrazione**
+========================================================
+
+.. toctree::
```

Comment:
Qui eviterei di creare un ulteriore livello di navigazione, invece suggerirei di inserire il testo direttamente in questo file rispettando le convenzioni per i titoli di sezioni e sotto-sezioni => https://docs.italia.it/italia/docs-italia/docs-italia-guide/it/bozza/appendice-2/titoli.html